README.md

```diff
@@ -1,2 +1,12 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+(a) The algorithm basically checks through the list to identify any duplications that may be present within the entries. If there are any, it will return as True. Then if not, it'll return as False.
+(b) If the list is in quadratic, it will need to run through the list either twice or multiple times depending on how many items are in the list. Adding more to the list will increase the amount of time taken. 
+(c) The algorithm is still correct, mainly because it still runs the list to see if there are any duplicates.
+(d) It makes it run through the list only once as it is checking each of the positions. Thus making it faster.  
```

Comment:
Key point to note: the original algorithm checks each pair of elements twice (a,b) and (b,a), whereas the new version checks each pair only once